pegasus/helper_modules/multiple_extname_file_utils.rb

```diff
@@ -0,0 +1,41 @@
+module MultipleExtnameFileUtils
+  # Some utilities for dealing with files with multiple extension names (IE,
+  # "foo.tar.gz")
+
+  def self.all_extnames(filename)
+    # Grab an array representing all extensions applied to the
+    # given file.
+    # IE, "foo.erb.md" -> [".erb", ".md"]
+
+    # Use a positive lookahead regex rather than a simple string match so the
+    # periods will be included in the resulting array, to better match the
+    # functionality of File.extname
+    File.basename(filename).split(/(?=\.)/).drop(1)
+  end
+
+  def self.file_has_any_extnames(filename, extnames)
+    # Returns true if any of the extensions used by the given file are in the
+    # given list of extension names
```

Comment:
Is this comment accurate?

Reply:
Yep! Note it's checking that the intersection of "extensions used by the given file" and "given list of extension names" is nonempty.